python/referer_parser/__init__.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, ref_url, curr_url=None, referers=REFERERS):
         self.known = ref_uri.scheme in {'http', 'https'}
         self.uri = ref_uri
 
-        if not self.known:
+        if not self.known or not ref_host:
```

Comment:
I would prefer this check be wrapped up into the self.known variable, such that `self.known == False` whenever ref_host is falsey.